File: CS-Examples/18_Tables/AllowBreakAcrossPages.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;

namespace AllowBreakAcrossPages
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create word document
			Document document = new Document();

			//Load the file from disk
			document.LoadFromFile(@"..\..\..\..\..\..\Data\AllowBreakAcrossPages.docx");

			//Get the first section
			Section section = document.Sections[0];

			//Get the first table
			Table table = section.Tables[0] as Table;

			//Loop through the table rows
			foreach (TableRow row in table.Rows)
			{
				//Allow break across pages
				row.RowFormat.IsBreakAcrossPages = true;
			}

			//Save the Word document
			string output = "AllowBreakAcrossPages_out.docx";
			document.SaveToFile(output, FileFormat.Docx2013);

			//Dispose the document
			document.Dispose();

            //Launch the file
            WordDocViewer(output);
        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/18_Tables/AllowBreakAcrossPages.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents

Namespace AllowBreakAcrossPages
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			'Create a Word document
			Dim document As New Document()

			'Load the file from disk
			document.LoadFromFile("..\..\..\..\..\..\Data\AllowBreakAcrossPages.docx")

			'Get the first section
			Dim section As Section = document.Sections(0)

			'Get the first table
			Dim table As Table = TryCast(section.Tables(0), Table)

			'Loop through the table rows
			For Each row As TableRow In table.Rows

				'Allow break across pages
				row.RowFormat.IsBreakAcrossPages = True
			Next row

			'Save the Word document
			Dim output As String = "AllowBreakAcrossPages_out.docx"
			document.SaveToFile(output, FileFormat.Docx2013)

			'Dispose the document
			document.Dispose()

			'Launch the file
			WordDocViewer(output)
		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
